lib/contact_searcher.rb

```diff
@@ -107,6 +105,12 @@ def api_where(selector=nil)
             end
           when 'first_name', 'last_name'
             self.new_selector[k] = v.is_a?(String)? Regexp.new(v,Regexp::IGNORECASE) : v
+          when 'tags'
+            cs = Tag.find(v).map{|t| t.contact_ids}.flatten
+            contacts_with_tags = cs.select{|c| cs.count(c) > 1}.uniq
+            andit({
+              :_id => {'$in' => contacts_with_tags}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.